Project.toml

```diff
@@ -23,6 +27,7 @@ OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
 SteadyStateDiffEq = "9672c7b4-1e72-59bd-8a11-6ac3964bc41f"
 StochasticDiffEq = "789caeaf-c7a9-5a7d-9973-96adeb23e2a0"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 
 [targets]
-test = ["Test", "OrdinaryDiffEq", "StochasticDiffEq", "SteadyStateDiffEq"]
+test = ["Test", "OrdinaryDiffEq", "StochasticDiffEq", "SteadyStateDiffEq", "Plots"]
```

Comment:
why?

Reply:
This is possibly me working with things I am not expert on, but as we require Plots in the tests, but no where else, I figure it belonged here.